### ISSUE TITLE
Error message not shown.

### ISSUE THREAD

**Issue by nasrin1748** [pyscript/pyscript#1965]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

Uploading subfolder which already exists doesn't show error msg like uploading subfile which already exists.Error msg should be shown for folder as well.

### What browsers are you seeing the problem on? (if applicable)

Microsoft Edge

### Additional Context

<img width="215" alt="n7" src="https://github.com/pyscript/pyscript/assets/79505893/f02d8d54-74f8-4395-b9ae-927ece19752c">

**Comment by WebReflection** [comment]
pinging @FabioRosado here as these bugs don't belong to this project.

**Comment by FabioRosado** [comment]
Thank you @nasrin1748  for opening this issue and @WebReflection  for pinging, I just passed this along to @alex-martinez  😄

**Comment by alex-martinez** [comment]
Hi @nasrin1748, this is expected because uploading a folder or subfolder that already exists isn't a conflict since the files in it will be merged into the existing folder/subfolder. 

For example, if I had `my-folder/subfolder/some-file.txt` in my project and I uploaded `my-folder/subfolder/README.md`, rather than prompt the user to overwrite the whole `my-folder` folder, the two are merged so we end up with `my-folder/subfolder/` containing `README.md` and `some-file.txt`. 

I felt this was an optimal experience, but please lmk if you feel otherwise.

**Comment by nasrin1748** [comment]
I had `folder/index.html` in my project and I uploaded `folder/index.html`,it shows error msg.
<img width="213" alt="1" src="https://github.com/pyscript/pyscript/assets/79505893/a3bfb6e8-04d2-46ba-a2b8-1986998ce8b5">

I had `folder/folder1` in my project and I uploaded `folder/folder1` ,it  does'nt show error msg.
It's more like enhancement.Instead of error message if it will ask to replace with existing folder or existing file it will be better approach i think.
